cylc/flow/scripts/install.py

```diff
@@ -156,6 +156,10 @@ def get_option_parser():
 
 @cli_function(get_option_parser)
 def main(parser, opts, flow_name=None, src=None):
+    install(parser, opts, flow_name, src)
+
+
+def install(parser, opts, flow_name=None, src=None):
```

Comment:
This doesn't appear to change anything, at least not without changing the entry point name as well - or am I missing something :thinking:

Reply:
It shouldn't change anything here. 
The purpose of this change is to allow the rose-stem script to import `install` without having to worry about the decorator, since it supplies it's own CLI options to install.